#
# Message containing all current poses.
#

std_msgs/Header header

geometry_msgs/Pose l_leg_pose
geometry_msgs/Pose r_leg_pose
geometry_msgs/Pose l_arm_pose
geometry_msgs/Pose r_arm_pose